js/modules/gomodule.go

```diff
@@ -25,10 +25,22 @@ func (gm *goModule) Instantiate(rt *sobek.Runtime) (sobek.CyclicModuleInstance,
 	if gm.exportedNames == nil {
 		named := mi.Exports().Named
 
-		gm.exportedNames = make([]string, len(named))
-		for name := range named {
-			gm.exportedNames = append(gm.exportedNames, name)
+		switch {
+		case len(named) == 0 && mi.Exports().Default != nil:
```

Comment:
I really do prefer using `len` as well instead of checking for `nil`. But in the other place we have this logic `toESModuleExports` it is specifically checking for `nil` so that if you do not want to use this functionality you can have `Named` be an empty list.

I very much doubt this is used, but will prefer to keep it consistent

Reply:
Sure, i've made the change in 241af938bc40e49c94787e05d10dd41958a7ecb0